node/request.js

```diff
@@ -43,7 +44,7 @@ function TChannelRequest(channel, options) {
     self.options = options;
     self.triedRemoteAddrs = {};
     self.outReqs = [];
-    self.timeout = self.options.timeout;
+    self.timeout = self.options.timeout || DEFAULT_TIMEOUT;
```

Comment:
there is another default timeout somewhere else of 5s. We should make the default timeout consistent across the board.